cluster-provision/k8s/1.31/manifests/kubeadm.conf

```diff
@@ -0,0 +1,54 @@
+apiVersion: kubeadm.k8s.io/v1beta3
```

Comment:
Throws a deprecation warning in the log: https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirtci/1220/check-provision-k8s-1.31/1813206378953576448#1:build-log.txt%3A4383